composer.json

```diff
@@ -14,6 +14,9 @@
         "issues": "https://www.mongodb.com/support",
         "security": "https://www.mongodb.com/security"
     },
+    "repositories": [
+        { "type": "vcs", "url": "https://github.com/GromNaN/mongo-php-builder.git" }
```

Comment:
To be removed once https://github.com/mongodb/mongo-php-builder/pull/65 is merged and tagged.